Detectors/MUON/MCH/Clustering/include/MCHClustering/ClusterConfig.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ClusterConfig.h
+/// \brief Clustering and fifting parameters
+/// \author Gilles Grasseau, Subatech
+
+#ifndef O2_MCH_CLUSTERCONFIG_H_
+#define O2_MCH_CLUSTERCONFIG_H_
+
+namespace o2
+{
+namespace mch
+{
+
+typedef int PadIdx_t;   // Pad index type
+typedef short Groups_t; // Groups/sub-cluster type
+typedef short Mask_t;   // To build mask
+
+struct ClusterConfig {
```

Comment:
For the record (not to be modified for this PR), when we'll get in production with this new clustering (not for this PR either, unfortunately ;-) ), this configuration should probably be integrated with the existing [MCHClusterizerParam](https://github.com/AliceO2Group/AliceO2/blob/dev/Detectors/MUON/MCH/Clustering/include/MCHClustering/ClusterizerParam.h) in order to be steerable "from the outside"

Reply:
The configuration of new algo is different from the old one but this configuration could inherit from the  ClusterizerParam or maybe better, having a subset of ClusterConfig.h steerable from outside (with the same variable name for common values).
Add in my long TODO list ...